src/mesh/generated/meshtastic/telemetry.pb.h

```diff
@@ -102,8 +102,10 @@ typedef enum _meshtastic_TelemetrySensorType {
     meshtastic_TelemetrySensorType_SEN5X = 43,
     /* TSL2561 light sensor */
     meshtastic_TelemetrySensorType_TSL2561 = 44,
+    /* Adafruit STEMMA Soil Sensor */
```

Comment:
Thanks for your work! Just so you know, these files are automatically generated and so the edits should go to the meshtastic/protobufs repo